aggregator/src/binaries/aggregator.rs

```diff
@@ -394,6 +394,11 @@ pub struct Config {
     /// [`TASK_AGGREGATOR_CACHE_DEFAULT_CAPACITY`]. You shouldn't normally have to specify this.
     #[serde(default)]
     pub task_cache_capacity: Option<u64>,
+
+    /// Experimental. Always advertise global HPKE keys instead of per-task HPKE keys. This will
+    /// become on by default in a future version of Janus.
+    #[serde(default)]
+    pub require_global_hpke_keys: Option<bool>,
```

Comment:
nit: is `None` the same as `Some(false)`? if so, for simplicity I think this should just be a `bool` which defaults to `false`, so that we only interpret a missing value once.